### PR TITLE
Fix installation of python via apt-get

### PR DESCRIPTION
**Pull request by dkulacz**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -145,5 +145,6 @@ RUN apt-get -y update && \
         build-essential \
         cmake \
         ninja-build \
-        python-minimal \
-    && rm -rf /var/lib/apt/lists/*
+        python2.7 \
```
Comment (darklukee) [inline Dockerfile:148]: just install python2, no need for links then.

**Review by darklukee — CHANGES_REQUESTED** [review]

File: Dockerfile (hunk 1 above)
Comment (dkulacz) [reply Dockerfile:148]: unfortunatelly there is no python2 package :(

Comment (dkulacz) [reply Dockerfile:148]: @darklukee any hint how to do it properly ?

Comment (darklukee) [reply Dockerfile:148]: ok, I see. python2 package is available on ubuntu 20.04, not 18.04 which is used here.
You could use update-alternatives tool, but I'm fine with link.

**Review by darklukee — APPROVED** [review]

File: Dockerfile (hunk 1 above)
Comment (kwesolowski) [reply Dockerfile:148]: lets keep the link and fix in #16

**Pull request merged**